File: Core python/Assignment/Assignment 3/q2.py
# Write a program to input any alphabet and check whether it is vowel or consonant.

# take input of alphabet
ch = input("enter the alphabet : ")

# create list for compare ch to vowel
list = ['a' , 'e' , 'i' , 'o' ,'u']

if(ch.lower() in list ):
    print(f'{ch} is vowel')
else:
    print(f'{ch} is consonant')

File: C/Assignments/Assignment 1/q2.c
// Write a program to check given 3 digit number is pallindrome or not.


void main(){
	
	//take input number
	int number , d1, d2, d3;
	printf("enter the number : ");
	scanf("%d" , &number);
	
	//calculate digit for reverse number
	int n = number ;
	
	d1 = n % 10;
	n = n / 10 ;
	
	d2 = n % 10 ;
	n = n / 10;
	
	d3 = n % 10;
	
	// # calculate  reverse number
	int reverse = (d1 * 100) + (d2 * 10) + d3 ;
	
	// check and print , number is palindrome or not
	if(number == reverse){
		printf("number is palindrome");
	
	}
	
	else {
		printf("number is not palindrome");
	}
	
	
}